### DISCUSSION TITLE
AI口碑-餐飲行銷🍽

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#81] [General]
**新聞內容：**
本土新創、一站式微網紅媒合平台Influenxio 圈圈科技推出業界首創、專為餐飲業設計的**AI全自動口碑行銷方案**，以訂閱制模式，由系統自動挑出合適人選並直接展開合作，滿足地域性口碑行銷需求，打造市場熱度，讓各類規模都能開啟 Instagram 微網紅行銷。
該公司以資料庫數據調查發現，一萬粉絲以下的微網紅因黏著度、互動率高，有9成5的品牌計畫以此做為宣傳、增加行銷預算。透過平台評估案件、自主報價與品牌開啟合作，簡化過去合作的繁複溝通，並將流程、報價等全透明化。
公司目前已創立三年、已協助近７,000個品牌產出累積逾三萬則 Instagram 貼文。
**新聞心得：**
我覺得這個平台對於宣傳個人開創又有獨特風格的餐飲小企業很友善，品牌知名度高的連鎖店自然有顧客上門，但一家餐飲品質很好的小店家是很需要去推廣自家商品及特色才能吸引人群，能透過合作讓自己的聲量變大，相信雙方都能有收穫，且透明化的資訊才比較能讓人放心！
**新聞來源：**
https://udn.com/news/story/7240/5828297
 
![image](https://user-images.githubusercontent.com/90883526/138582905-20e198b9-3eb8-4f30-b43d-a25363b63dbf.png)

**Comment by chia-kai-chang** [comment]
這種商業模式我第一次聽過，好新穎
有點像是之前的104家教，或104接案，變成廣告交換或是網紅業配報價
對於訂閱數還不高的奈米網紅，可以有一個接案的管道

對於圈圈科技來說，I站上的會員就是勞動力來源，一定要使用網路爬蟲把網紅的各項數據撈出來定期更新，才能與業者做有效媒合

這個數據為王的時代，相較於104家教、104接案，I站的勞動力目標好明顯阿~ 並且有源源不絕的勞動力的感覺~

**Reply by bonny123865** [reply]
真的真的，而且一定很吸引餐飲業者想合作！
